### PR TITLE
add  etcd to @configu/integrations

### LINKED ISSUE #514: Migrate `EtcdConfigStore` to `@configu/integrations`
### Suggestion

Migrate the `EtcdConfigStore` from its current setup to the new `@configu/integrations` module as part of the major restructuring. The goal is to move all third-party integrations, including stores, expressions, and libraries, to `@configu/integrations`.

### Motivation

This migration is part of a larger effort to better organize and modularize the product. Moving the `EtcdConfigStore` will align with the new structure, making the system cleaner and more maintainable.

### Context

You can refer to the already migrated stores for guidance:  
[Migrated stores in integrations](https://github.com/configu/configu/tree/main/packages/integrations/stores).

Base stores implementations from `@configu/ts` can be found here:  
[Base stores in TS](https://github.com/configu/configu/tree/52cee9c41fb03addc4c0983028e37df42945f5b7/packages/ts/src/stores).

The old node-based stores were previously in `@configu/node`:  
[Node stores](https://github.com/configu/configu/tree/52cee9c41fb03addc4c0983028e37df42945f5b7/packages/node/src/stores).

The old browser-based stores were previously in `@configu/browser`:  
[Browser stores](https://github.com/configu/configu/tree/52cee9c41fb03addc4c0983028e37df42945f5b7/packages/browser/src/stores).

### PR DESCRIPTION
**Pull request by su-shubham**
Closes #514

### CODE REVIEW

File: packages/integrations/stores/etcd/package.json [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+  ],
+  "scripts": {
+    "build": "tsup"
+  },
+  "dependencies": {
+    "@configu/sdk": "workspace:*",
+    "@configu/integrations": "workspace:*"
+  },
+  "devDependencies": {
+    "etcd3": "1.1.2",
```
Comment (davidmeirlevy) [inline packages/integrations/stores/etcd/package.json:29]: should be a dependency, not dev.

**Review by davidmeirlevy — CHANGES_REQUESTED** [review]
@su-shubham thanks for your PR!

just a minor fix, and we can merge it! 👏

**Review by davidmeirlevy — APPROVED** [review]
LGTM

**Comment by davidmeirlevy** [comment]
@su-shubham thanks for your contribution 🚀

**Comment by pelegpor** [comment]
Hi @su-shubham ,

Here are the details for claiming your Hacktoberfest prizes! 🎉 Check out [#465  (comment)](https://github.com/configu/configu/issues/465#issuecomment-2498550259) for more information or simply fill out the form here: [Claim Your Swag](https://share-eu1.hsforms.com/2SVc5VKY3Thi04Bvjh17WKw2e62o0).  

Remember, you only need to fill out the form once for all your PRs. 😊

**Pull request merged**